### PR TITLE
Fx/images, tailwindconfig and consistency

### PR DESCRIPTION
**Pull request by devfabien**
# PR Description

This Pull Request includes the changes in the structure of the codes like the tailwind configuration, adding external css file removing unnecessary files,....

## Description of tasks that were expected to be completed

List all the tasks and sub-tasks that were assigned to you Make sure you check every completed task. list them in this style

- [x] Removing google links 
- [x] Reconfiguration of colors and font names
- [x] Removing unnecessary documents and files
- [x] Creating an external css file to hold inline styles
- [x] Adding meaningful alt attributes texts
- [x] Keep consistency between px and rem
- [x] Importing svgs 

## Description of tasks that were expected to be completed and that are not yet completed
- [x] None

## How this may be tested?

Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

1. run `npm i` to install all required dependencies
2. run `npm run build`
3. run `npm run start`

## Screenshots (If appropriate/applicable)

Provide screenshots related to your PR in case it's appropriate and applicable

## Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

**Review by kareraolivier — APPROVED** [review]
great work

File: src/sections/founder.html [hunk 1]
```diff
@@ -1,129 +1,155 @@
...
+      />
+      <div
+        class="flex flex-wrap py-8 bg-gradient-to-r from-greenn/50 via-blue_990 lg:w-4/5"
+      >
+        <div class="px-4 lg:px-12 md:py-4 lg:w-3/5 z-10">
+          <h1 class="pb-4 font-bold text-lg">
+            Ein
+            <span class="relative">
+              stetig wachsender
+              <svg
```
Comment (kunda4) [inline src/sections/founder.html]: for making code more readable and maintainable import this svg too

File: src/sections/hero.html [hunk 2]
```diff
@@ -1,76 +1,64 @@
...
-
+  <div class="h-[50rem] bg-gradient-to-r from-blue_990 from-40% to-greenn/50">
+    <nav class="flex flex-row py-2 px-4 lg:px-20 justify-between items-center">
+      <div>
+        <img src="./src/assets/Logo.svg" alt="Ojemba logo" class="py-1 w-3/4">
+      </div>
+      <div class="flex justify-between gap-5 lg:gap-20 items-center backdrop:">
+        <span class="lg:hidden cursor-pointer">
+          <?xml version="1.0" encoding="utf-8"?>
+          <svg
```
Comment (kunda4) [inline src/sections/hero.html]: Import this one too

File: src/sections/managedTeam.html [hunk 3]
```diff
@@ -1,78 +1,151 @@
...
+          tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
+          voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
+          Stet clita kasd gubergren, no sea takimata sanctus est L
+        </p>
+      </div>
+      <div class="lg:w-3/4 my-5 md:flex gap-4">
+        <h2 class="font-bold font-Raleway w-fit whitespace-nowrap">
+          So
+          <span class="relative"
+            >arbeitet<svg
```
Comment (kunda4) [inline src/sections/managedTeam.html]: Import this one too

File: src/sections/managedTeam.html [hunk 4]
```diff
@@ -1,78 +1,151 @@
...
+          class="py-12 w-full md:flex justify-between gap-9 font-Montserrat"
+          id="accordion1"
+        ></div>
+      </div>
+      <div class="lg:w-3/4 my-3 py-5 gap-4">
+        <h2 class="font-bold font-Raleway">
+          Ihre
+          <span class="relative"
+            >Vorteile
+            <svg
```
Comment (kunda4) [inline src/sections/managedTeam.html]: and this one

File: src/sections/managedTeam.html [hunk 5]
```diff
@@ -1,78 +1,151 @@
...
+      <div class="ml-14 md:flex hidden justify-center items-center">
+        <img src="./src/assets/maibornw.webp" class="h-2/12" alt="MaibornWolff logo"/>
+      </div>
+    </div>
+    <div
+      class="border-b-4 border-emerald-800 flex justify-between py-2 font-bold para text-sm"
+    >
+      <h2>African Team Box</h2>
+      <h2 class="w-fit h-fit cursor-pointer">
+        <svg
```
Comment (kunda4) [inline src/sections/managedTeam.html:84]: import this one too

**Review by kunda4 — CHANGES_REQUESTED** [review]
Hello @devfabien you did a great job but for making code more readable and maintainable it will be better if you import all svg for making code more readable

**Review by kunda4 — APPROVED** [review]
good job

**Pull request merged**